app/src/sharedTest/java/org/oppia/app/home/RecentlyPlayedFragmentTest.kt

```diff
@@ -66,9 +120,13 @@ class RecentlyPlayedFragmentTest {
   }
 
   @Test
-  @Ignore("Reenable once OngoingStoryList in domain layer is updated.") // TODO(#735): Reenable once OngoingStoryList is fetched correctly.
   fun testRecentlyPlayedTestActivity_recyclerViewItem0_doesNotShowSectionDivider() {
-    ActivityScenario.launch(RecentlyPlayedFragmentTestActivity::class.java).use {
+    ActivityScenario.launch<RecentlyPlayedActivity>(createRecentlyPlayedActivityIntent(internalProfileId)).use {
+      onView(withId(R.id.ongoing_story_recycler_view)).perform(
+        scrollToPosition<RecyclerView.ViewHolder>(
```

Comment:
Indentation is incorrect. `ScrollToPosition `method can be written in single line. Please correct it here and also at other similar places in this file.

Reply:
Indentation is something which we cannot confirm that it should follow  one single pattern because of android studio. The only thing that we can make sure is that the indentation in one single file is consistent. I have done that. Thanks.